### PR TITLE
ADD Documentation for AMQP

### PR DESCRIPTION
**Pull request by dmoranj**

### CODE REVIEW

File: docs/usermanual.md [hunk 1]
```diff
@@ -163,6 +163,28 @@ The result of the command must be reported in the following topic:
 ```
 The command execution and command reporting payload format is specified under the Ultralight 2.0 Commands Syntax, above.
 
+#### AMQP
+
+AMQP stands for Advance Message Queuing protocol, and is one of the most popular protocols for message-queue systems.
+Altough the protocol itself is software independent and allows for a great architectural flexibility, this transport
```
Comment (AlvaroVega) [inline docs/usermanual.md]: typo: Altough -> Although

**Comment by AlvaroVega** [comment]
LGTM

File: docs/usermanual.md [hunk 2]
```diff
@@ -163,6 +161,28 @@ The result of the command must be reported in the following topic:
 ```
 The command execution and command reporting payload format is specified under the Ultralight 2.0 Commands Syntax, above.
 
+#### AMQP
+
+AMQP stands for Advance Message Queuing protocol, and is one of the most popular protocols for message-queue systems.
```
Comment (frbattid) [inline docs/usermanual.md]: protocol --> Protocol
Link to AMQP home would be nice

**Comment by frbattid** [comment]
LGTM

**Pull request merged**